packages/neos-ui/src/Containers/Drawer/style.css

```diff
@@ -1,19 +1,33 @@
 .drawer {
     position: fixed;
-    top: 41px;
+    display: flex;
+    flex-flow: column;
+    flex-shrink: 0;
+    justify-content: flex-start;
+    top: calc(var(--spacing-GoldenUnit) + 1px);
     left: 0;
     z-index: var(--zIndex-Drawer-Context);
     height: 100%;
```

Comment:
this needs to be lower because of the top bar, otherwise you could'nt scroll down enough to see the version number when there are many items. (I think something like `100% - goldenUnit` or smth)
![screenshot-2018-6-26 neos demo site](https://user-images.githubusercontent.com/12071529/41905052-51891932-793a-11e8-85dd-08c1163b0c4c.png)